app/boards/arm/nrfmicro/Kconfig.defconfig

```
@@ -0,0 +1,33 @@
+# Electronut Labs Papyr board configuration
+
+# Copyright (c) 2020 Pete Johanson, Okke Formsma, joric
+# SPDX-License-Identifier: MIT
+
+if BOARD_NRFMICRO
```

This needs to be updated to:

```suggestion
if BOARD_NRFMICRO_11 || BOARD_NRFMICRO_11_FLIPPED | BOARD_NRFMICRO_13
```

And that should make the defaults be honored properly.